UnityProject/Assets/Scripts/Systems/Voting/VotingManager.cs

```diff
@@ -305,13 +305,13 @@ private void FinishVote()
 
 	private string CountAmountString()
 	{
-		return $"{ForVoteCount()} / {PlayerList.Instance.AllPlayers.Count}";
+		return $"{TotalVotes()} / {PlayerList.Instance.AllPlayers.Count}";
 	}
 
 	/// <summary>
 	/// Gets number of coutns
 	/// </summary>
-	private int ForVoteCount()
+	private int TotalVotes()
```

Comment:
why make this a method?

Reply:
I wanted to check if it was null but forgot to add that bit, fixed now.

Reply:
actually does votes.Clear() even turn it null?

Reply:
no, clear only cleans the elements that the list contains.